libraries/chain/include/eosio/chain/wast_to_wasm.hpp

```diff
@@ -1,6 +1,7 @@
 #pragma once
 #include <vector>
 #include <string>
+#include <stdint.h>
```

Comment:
Why not `<cstdint>` ?

Reply:
I was trying to simply add the missing header: not refactor code in any way. In this case there is no `std` usage in here so `stdint.h` seemed like the more correct choice.